mediaretry.go

```diff
@@ -9,11 +9,12 @@ package whatsmeow
 import (
 	"fmt"
 
-	"go.mau.fi/util/random"
 	"google.golang.org/protobuf/proto"
 
+	"go.mau.fi/util/random"
+
```

Comment:
`go.mau.fi/util` is an external module from whatsmeow's perspective, so it should be sorted with other external non-stdlib modules. Imports should always be sorted into stdlib, external (all other modules) and internal (current module)